collector/src/main.rs

```diff
@@ -188,6 +189,7 @@ fn bench(
     let mut measure_and_record =
         |benchmark_name: &BenchmarkName,
          benchmark_supports_stable: bool,
+         benchmark_category: &str,
```

Comment:
This is being passed as &str, but it seems like it should be passed as the enum -- that probably means moving the enum definition to database/src/lib.rs?

Reply:
Well, it was my intention not to put this into the database crate. In the database it will just be a string, so that's the interface that I wanted to use for it. This way we can change the representation in the collector and it won't affect the database.

Reply:
I would prefer to intentionally update the database -- yes, we're storing it as a string under the hood, but all Rust code should largely deal with the enum and convert to it at boundary layers with disk (i.e., benchmarks directory) or db (postgres/sqlite).

Reply:
The string from the DB is passed verbatim to the frontend though, it's not being handled in Rust at all at the database/site crate. But I agree that it would be nicer, I can change it.